js/auth.js

```diff
@@ -0,0 +1,85 @@
+const API_BASE_URL = 'https://api.noroff.dev/api/v1';
```

Comment:
I will advice you extract this to utility file because you will need this in other files

Reply:
yes, very true. done!